Data_Sender.cpp

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <math.h>
+#include "Data_Sender.hpp"
+
+void readSenderDataFromFile(float *Temperature, int *StateOfCharge,char *inputFileName)
```

Comment:
StateofCharge variable could have been declared as float to cover wide range of values